src/EnergyPlus/WindowComplexManager.cc

```diff
@@ -2838,13 +2838,13 @@ namespace WindowComplexManager {
         nmix(nlayer + 1) = 1;      // pure air on indoor side
 
         // Simon: feed gas coefficients with air.  This is necessary for tarcog because it is used on indoor and outdoor sides
-        GasType = DataComplexFenestration::GasCoeffs::Air;
-        wght(iprop(1, 1)) = GasWght(static_cast<int>(GasType));
-        gama(iprop(1, 1)) = GasSpecificHeatRatio(static_cast<int>(GasType));
+        GasType = static_cast<int>(DataComplexFenestration::GasCoeffs::Air);
+        wght(iprop(1, 1)) = GasWght[GasType - 1];
+        gama(iprop(1, 1)) = GasSpecificHeatRatio[GasType - 1];
         for (ICoeff = 1; ICoeff <= 3; ++ICoeff) {
-            gcon(ICoeff, iprop(1, 1)) = GasCoeffsCon(ICoeff, static_cast<int>(GasType));
-            gvis(ICoeff, iprop(1, 1)) = GasCoeffsVis(ICoeff, static_cast<int>(GasType));
-            gcp(ICoeff, iprop(1, 1)) = GasCoeffsCp(ICoeff, static_cast<int>(GasType));
+            gcon(ICoeff, iprop(1, 1)) = GasCoeffsCon[ICoeff - 1][GasType - 1];
+            gvis(ICoeff, iprop(1, 1)) = GasCoeffsVis[ICoeff - 1][GasType - 1];
+            gcp(ICoeff, iprop(1, 1)) = GasCoeffsCp[ICoeff - 1][GasType - 1];
```

Comment:
I just made `GasType` and int here and went with one `static_cast` instead of five.